src/main/java/org/torusresearch/torusutils/apis/APIUtils.java

```diff
@@ -77,6 +73,7 @@ public static CompletableFuture<String> post(String url, String data, Boolean us
         return _post(url, data, new Header[0], useApiKey);
     }
 
+    @SuppressWarnings("unused")
     public static CompletableFuture<String> post(String url, String data, Header[] headers, Boolean useApiKey) {
```

Comment:
Just a note here for a future PR.

We should replace CompletableFuture<> with Future<>. The former is not interruptable or cancellable.